lgc/state/PipelineState.cpp

```diff
@@ -726,6 +872,9 @@ void PipelineState::setColorExportState(ArrayRef<ColorExportFormat> formats, con
 //
 // @param location : Export location
 const ColorExportFormat &PipelineState::getColorExportFormat(unsigned location) {
+  if (m_colorExportState.dualSourceBlendEnable)
+    location = 0;
+
```

Comment:
This is the fixed mentioned in the commit message. Previously, the callers had to set the location to 0 when dualSourceBlenEnable was set -- that code is deleted now.